epics/ca.py

```diff
@@ -1031,13 +1031,15 @@ def array_cast(data, count, ntype, use_numpy):
             else:
                 out = numpy.ctypeslib.as_array(memcopy(data))
         else:
-            out = memcopy(data)
+            out = list(data)
         return out
 
-    def unpack(data, count, ntype, use_numpy):
+    def unpack(data, count, ntype, use_numpy, elem_count):
         "simple, native data type"
         if data is None:
             return None
+        elif ntype == dbr.CHAR and elem_count > 1:
```

Comment:
Proposed solution was `if count == 1 and ntype == dbr.CHAR and ncount > 1:`, but I believe we always want to get an array back if it's a char waveform (so a value of '\0\0' with a count of 2 would give back an array too)